cmd/kpod/kill.go

```diff
@@ -0,0 +1,65 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/kubernetes-incubator/cri-o/libkpod"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+	"os"
+)
+
+var (
+	killFlags = []cli.Flag{
+		cli.StringFlag{
+			Name:  "signal, s",
+			Usage: "Signal to send to the container",
+			Value: "KILL",
```

Comment:
Nit: `KILL` -> `SIGKILL` for consistency with the description

Reply:
fixed